dask/sharedict.py

```diff
@@ -53,7 +53,10 @@ def __init__(self, dicts=None, dependencies=None):
         self.dicts = dicts or dict()
         self.dependencies = dependencies or dict()
 
-    def update_with_key(self, arg, key=None):
+        assert set(self.dependencies) == set(self.dicts)
+        # assert all(dep in self.dicts for v in self.dependencies.values() for dep in v)
```

Comment:
Any value in keeping this comment around?

Reply:
removed